org.aardappel.TreeSheets.json

```diff
@@ -0,0 +1,100 @@
+{
+    "app-id": "org.aardappel.TreeSheets",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/treesheets",
+    "branch": "master",
+    "tags": ["master", "notes", "wxWidgets"],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        },
+        "build-args": [
+            "--share=network"
+        ]
+    },
+    "cleanup": [
+        "/include",
+        "/tests",
+        "/man",
+        "/share/man",
+        "treesheets.desktop",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "wxWidgets",
+            "config-opts": [
+                "--enable-unicode"
+            ],
+            "cxxflags": [
```

Comment:
This should print a warning about an invalid property. This goes in `build-options`, see `man flatpak-manifest`